File: src/taburet/accounting/_design/accounting/views/transactions/map.js
function(doc) {
	if ( doc.doc_type == 'Transaction' ) {
		doc.from_acc.forEach(function(e) {
			emit([e, doc.date[0], doc.date[1], doc.date[2]], null)
		})
		doc.to_acc.forEach(function(e) {
			emit([e, doc.date[0], doc.date[1], doc.date[2]], null)
		})
	}
}

File: taburet/transactions/_design/transactions/views/balance_for_account/map.erl
fun({Doc}) ->
	case proplists:get_value(<<"doc_type">>, Doc) of
		<<"Transaction">> ->
			Amount = proplists:get_value(<<"amount">>, Doc),
			[Year, Month, Day|_] = proplists:get_value(<<"date">>, Doc), 
			
			lists:foreach(
				fun(AccId)-> Emit([AccId, Year, Month, Day], {[{debet, 0.0}, {kredit, Amount}]}) end,
				proplists:get_value(<<"from_acc">>, Doc)
			),
			
			lists:foreach(
				fun(AccId)-> Emit([AccId, Year, Month, Day], {[{debet, Amount}, {kredit, 0.0}]}) end,
				proplists:get_value(<<"to_acc">>, Doc)
			);
		_ -> ok
	end
end.
